src/main/resources/view/MainWindow.fxml

```diff
@@ -1,59 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <?import java.net.URL?>
-<?import javafx.geometry.Insets?>
 <?import javafx.scene.Scene?>
-<?import javafx.scene.control.Menu?>
-<?import javafx.scene.control.MenuBar?>
-<?import javafx.scene.control.MenuItem?>
-<?import javafx.scene.control.SplitPane?>
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
 
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
-         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+         title="SplitWiser" minWidth="384" maxWidth="384" minHeight="700" maxHeight="700" onCloseRequest="#handleExit">
```

Comment:
Quick question, is there really a reason on why we are changing the window size to a mobile form factor? Its currently not a mobile app, and I feel like the window can be slightly larger to display more info. I can see stuff like view Activity needing more space, and we already have pretty long error messages that require users to scroll up/down to see the full error message.

Reply:
Oh well, I guess its a small change that can be done quickly in the future. Guess we are sticking with this change

Reply:
Have increased the app width to give us more canvas space to mess around with stuff :)